app/javascript/src/components/Common/Navbar/index.jsx

```diff
@@ -1,54 +1,89 @@
 import React from "react";
 
-import { Toastr } from "neetoui";
+import { UserCircle, NeetoInsights } from "@bigbinary/neeto-icons";
+import { Sidebar } from "@bigbinary/neetoui/v2/layouts";
 import { withRouter } from "react-router-dom";
 
-import authenticationApi from "apis/authentication";
-import { resetAuthTokens } from "apis/axios";
-import { useAuthDispatch } from "contexts/auth";
-
-import AccountDropdown from "./AccountDropdown";
-import NavItem from "./NavItem";
-
 const NavBar = () => {
-  const authDispatch = useAuthDispatch();
-  const handleLogout = async () => {
-    try {
-      await authenticationApi.logout();
-      authDispatch({ type: "LOGOUT" });
-      resetAuthTokens();
-      window.location.href = "/";
-    } catch (error) {
-      Toastr.error(error);
-    }
-  };
-
   return (
-    <div className="bg-gray-100 nh-sidebar" key="sidebar">
-      <div className="nh-logo">
-        <div className="flex items-center justify-center w-8 h-8 rounded-md">
-          <i className="text-purple-500 ri-flashlight-fill ri-2x" />
-        </div>
-      </div>
-      <div className="flex flex-col items-center justify-between w-full h-full">
-        <div className="flex flex-col items-center justify-start w-full pt-4">
-          <NavItem title="Notes" link="/notes" icon="ri-file-text-line" />
-          <NavItem
-            title="Settings"
-            link="/settings"
-            icon="ri-settings-2-line"
-            subLinks={[
-              { title: "Change password", link: "/my/password/edit" },
-              { title: "My Profile", link: "/my/profile" }
-            ]}
-          />
-        </div>
-        <div className="mb-4">
-          <AccountDropdown handleLogout={handleLogout} />
-        </div>
-      </div>
+    <div className="flex flex-row items-start justify-start">
+      <Sidebar
+        isCollapsed={true}
+        organizationInfo={{}}
+        navLinks={[
+          {
+            label: "Notes",
+            to: "/notes",
+            icon: () => NotesIcon
+          },
+          {
+            label: "Contacts",
+            to: "/login",
+            icon: () => <UserCircle color="#1e1e20" size={24} />
+          },
+          {
+            label: "Settings",
+            to: "/my/profile",
+            icon: () => <NeetoInsights color="#1e1e20" size={24} />
+          }
+        ]}
```

Comment:
Please move this to a `constants.js` file. 
Also, while exporting constants from an external file use the following convention 
`export const NAV_LINKS`